<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI3DControlPadSetting</class>
 <widget class="QDialog" name="UI3DControlPadSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>414</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>3D Control Pad Configuration</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/actions/3dcontroller.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tbUp">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>145</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Up</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbX">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>120</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>X</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbY">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>105</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Y</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbZ">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Z</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbC">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>135</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>C</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>145</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>B</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbA">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>160</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>A</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbStart">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbRightTrigger">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Right trigger</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbLeftTrigger">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Left trigger</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbDown">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>210</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Down</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbRight">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>177</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Right</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbLeft">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>177</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Left</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lInfos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="padButtons">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tbAxis2Up">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>60</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Up</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbAxis1Left">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>90</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Left</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbAxis2Down">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>120</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Down</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbAxis1Right">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>90</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Right</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>padButtons</sender>
   <signal>rejected()</signal>
   <receiver>UI3DControlPadSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
